client/my-sites/term-selector/no-results.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import { localize } from 'i18n-calypso';
+
+class TermSelectorNoResults extends PureComponent {
+	constructor( props ) {
+		super( props );
+	}
+
+	render() {
+		const { createLink, translate } = this.props;
+		let createMessage;
+		let noResultsMessage = translate( 'No results. Please try a different search.' );
```

Comment:
Now that I mention it, we should really turn on the [ESLint `prefer-const` rule](http://eslint.org/docs/rules/prefer-const).